packages/Python/lldbsuite/test/lang/swift/clangimporter/bridging_header_headermap/Makefile

```diff
@@ -1,12 +1,12 @@
 LEVEL = ../../../../make
-SRCDIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 SWIFT_OBJC_INTEROP := 1
-USESWIFTDRIVER := 1
-# This Makefile overwrites the default rules.
+SWIFT_SOURCES := main.swift
 MAKE_DSYM := NO
-EXE :=
+LD_EXTRAS := -ldylib -L.
+SWIFTFLAGS_EXTRAS = -import-objc-header $(SRCDIR)/bridging-header.h \
```

Comment:
Please add an `-enable-objc-interop` to the flags to ensure that this works on all targets

Reply:
This test is marked as `@skipUnlessDarwin`. Which is probably why the old version didn't bother with it either. I think porting the test to Linux or other systems would be great, but I'd like to keep this PR as targeted as possible. It's already pretty big :-)